go-dlv.el

```diff
@@ -131,6 +132,19 @@ and source-file directory for your debugger."
   (setq paragraph-start comint-prompt-regexp)
   (run-hooks 'go-dlv-mode-hook))
 
+;;;###autoload
+(defun dlv-current-test ()
```

Comment:
I think this function might be generalized a bit, by putting a breakpoint at the current `file:line`. If test-name exists, you can call dlv with `-test.run` as you do, and if there isn't, you would just call `dlv debug`.

This way, this function would work everywhere, not just in tests. Thoughts?